secp256k1-sys/src/lib.rs

```diff
@@ -680,6 +680,15 @@ impl<T> CPtr for [T] {
 
 #[cfg(fuzzing)]
 mod fuzz_dummy {
+    /// Serialization logic:
+    /// If pk is created from sk keypair:
+    ///     - It is serialized with prefix 0x02: sk || [0xaa;32]
+    /// If pk is created from from slice:
+    ///     - 0x02||pk_bytes -> pk_bytes||[0xaa;32]
+    ///     - 0x03||pk_bytes -> pk_bytes||[0xbb;32]
+    ///     - 0x04||pk_bytes -> pk_bytes
+    /// This leaves the room for collision between compressed and uncompressed pks,
```

Comment:
Wait, maybe I'm confused by this. What is the collision here and why would it ever be an issue?

Reply:
Never mind, this is not an issue. But what I meant by collision is the following
```
let pk = PublicKey::from_str("0x04<32bytes><0xaa;32>"); // Should be considered uncompressed, but is considered as compressed. 
```

Reply:
Public keys don't, themselves, have any concept of "compressed" or not? That's purely a serialization-time difference, no?